### PR TITLE
General improvements

### PR DESCRIPTION
**Pull request by yUnoSTAHP**
- upgrade to .net 4.5.1
- improvements to the viewer (log viewer, refresh progressbar)
- additional configuration for jobs : max parallel jobs

### CODE REVIEW

File: src/NetDist.Server/HandlerInstance.cs [hunk 1]
```diff
@@ -328,8 +366,12 @@ public bool ReceivedResult(JobResult result)
         private void SetFailed()
         {
             Stop();
-            NextStartTime = null;
-            _handlerState = HandlerState.Failed;
+            // TODO DF activate
```
Comment (Roemer) [inline src/NetDist.Server/HandlerInstance.cs]: Please remove the todo

File: src/WpfClient/App.config [hunk 2]
```diff
@@ -1,11 +1,11 @@
-﻿<?xml version="1.0" encoding="utf-8" ?>
+<?xml version="1.0" encoding="utf-8"?>
 <configuration>
   <startup>
-    <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.5" />
+    <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.5.1"/>
```
Comment (Roemer) [inline src/WpfClient/App.config:4]: Why not directly to 4.6.2?

File: src/WpfClient/WpfClient.csproj [hunk 3]
```diff
@@ -9,29 +9,32 @@
...
-    <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v4.5.1</TargetFrameworkVersion>
     <FileAlignment>512</FileAlignment>
     <ProjectTypeGuids>{60dc8134-eba5-43b8-bcc9-bb4bc16c2548};{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}</ProjectTypeGuids>
     <WarningLevel>4</WarningLevel>
+    <TargetFrameworkProfile />
   </PropertyGroup>
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
-    <PlatformTarget>AnyCPU</PlatformTarget>
+    <PlatformTarget>x64</PlatformTarget>
```
Comment (Roemer) [inline src/WpfClient/WpfClient.csproj]: Why this change?

**Review by Roemer — CHANGES_REQUESTED** [review]

File: src/WpfClient/App.config (hunk 2 above)
Comment (yUnoSTAHP) [reply src/WpfClient/App.config:4]: My company doesn't allow to install 4.6.2 ;)

File: src/NetDist.Server/HandlerInstance.cs (hunk 1 above)
Comment (yUnoSTAHP) [reply src/NetDist.Server/HandlerInstance.cs]: sure , stupid me

File: src/WpfClient/WpfClient.csproj (hunk 3 above)
Comment (yUnoSTAHP) [reply src/WpfClient/WpfClient.csproj]: My computer somehow preferred to build the 32bit version, this change was just to make sure to get the 64bit. I'll change it back.

**Review by Roemer — APPROVED** [review]
